cypress/integration/languages.spec.ts

```diff
@@ -0,0 +1,64 @@
+describe("AutoComplete Component", () => {
+  beforeEach(() => {
+    cy.visit("/languages?");
+  });
+
+  it("should display the suggestion list", () => {
+    getDataCY("autocomplete-input")
+      .type("e")
+      .then(() => {
+        getDataCY("autocomplete-suggestions").its("length").should("eq", 3);
+      });
+  });
+
+  it("should not display the suggestion list if gibberish is entered", () => {
+    getDataCY("autocomplete-input")
+      .type("a")
+      .then(() => {
+        getDataCY("autocomplete-suggestions").should("not.be.visible");
+      });
+  });
+
+  it("should able to select the item in the list", () => {
+    getDataCY("autocomplete-input")
+      .type("e")
+      .then(() => {
+        getDataCY("autocomplete-suggestions").first().click();
+        getDataCY("autocomplete-input").should("have.value", "english");
+      });
+  });
+
+  it("should able to select the item in the list with down and enter key", () => {
+    getDataCY("autocomplete-input")
+      .type("e")
+      .type("{downArrow}")
+      .type("{downArrow}")
+      .type("{enter}")
+      .should("have.value", "enn");
+  });
+
+  it("should able to select the item in the list with up and enter key", () => {
+    getDataCY("autocomplete-input")
+      .type("e")
+      .type("{upArrow}")
+      .type("{upArrow}")
+      .type("{enter}")
+      .should("have.value", "enn");
+  });
+
+  it("should display the correct BCP47 subtext", () => {
+    getDataCY("autocomplete-input")
+      .type("e")
+      .then(() => {
+        getDataCY("autocomplete-suggestions").first().click();
+        getDataCY("autocomplete-subtext").should(
+          "have.text",
+          "BCP47Tag:\n    en"
+        );
```

Comment:
The assertion as written is brittle, as if so much as a space or newline is added or deleted, it breaks this test. How about?

```suggestion
        getDataCY("autocomplete-subtext").should("contain", "en");
```

If I understand correctly, `contain` uses the `.contain(text)` assertion from [chai-jquery](https://github.com/chaijs/chai-jquery#containtext).

Alternatively, you could use a regular expression, but it's a bit more annoying to implement:

```suggestion
        getDataCY("autocomplete-subtext")
          .invoke("text")
          .should("match", /BCP-?47\s?Tag:\s+en\b/);
```